apps/api/src/courses/api/courses.controller.ts

```diff
@@ -49,6 +52,7 @@ export class CoursesController {
     @Query("page") page: number,
     @Query("perPage") perPage: number,
     @Query("sort") sort: SortCourseFieldsOptions,
+    @Query("userId") userId: string,
```

Comment:
```suggestion
    @CurrentUser("userId") currentUserId: string,
```
Api cannot rely on information from FE regarding userId.